src/tokenize_cleanup.cpp

```diff
@@ -128,6 +128,42 @@ void tokenize_cleanup(void)
     * Since [] is expected to be TSQUARE for the 'operator', we need to make
     * this change in the first pass.
     */
+   if (cpd.settings[UO_mod_strict_ASCII].b)
+   {
```

Comment:
I am not entirely sure why you need that option. Do you guys plan to merge that into upstream?

Reply:
its unity requirement, if upstream owners are interested they can take